APISpecification.yaml

```diff
@@ -366,71 +420,56 @@ components:
           type: boolean
         assembly:
           $ref: '#/components/schemas/AssemblyInGenome'
-        coding_genes_count:
-          type: number
-          example: 20446
-        contig_n50:
-          nullable: true
-          type: number
-          example: 56413054
-        has_variation:
-          type: boolean
-        has_regulation:
-          type: boolean
-        annotation_provider:
+        assembly_provider:
+          $ref: '#/components/schemas/AssemblyProvider'
+        assembly_level:
+          type: string
+          example: complete genome
+        assembly_date:
           type: string
-          example: Ensembl
+          description: ISO date string in the YYYY-MM format
+          example: "2019-02"
+        annotation_provider:
+          $ref: '#/components/schemas/AnnotationProvider'
         annotation_method:
           type: string
-          example: Full genebuild
-        rank:
+          example: import
+        annotation_version:
+          type: string
+          example: GENCODE 34
           nullable: true
+        annotation_date:
+          type: string
+          description: ISO date string in the YYYY-MM format
+          example: "2019-02"
+        image_url:
+          type: string
+          description: Link to species image, if present
+          example: http://beta.ensembl.org/static/genome_images/homo_sapiens_38.svg
+          nullable: true
+        related_genomes_count:
           type: number
-          example: 1
+          description: The number of genomes related to the current one via species taxonomy id.
+            May just mean the number of alternative assemblies; but may also include genomes
+            that have the same assembly, but different annotations.
```

Comment:
Here's a puzzle: what should the value of this field be for genomes whose related species (identified by the `species_taxonomy_id` field) does not have any genomes other than this one. Should it be `0`, because there are no other related genomes, or `1`, because the species with this `species_taxonomy_id` contains this current genome?

Personally, I would prefer `0`; but consider also the number coming from ensembl-search-hub during lookup by species taxonomy id.